setup.cfg

```diff
@@ -61,6 +61,9 @@ test =
 docs =
     sphinx-rtd-theme
     sphinx-astropy
+roman =
+    roman_datamodels @ git+https://github.com/spacetelescope/roman_datamodels.git
+    rad @ git+https://github.com/spacetelescope/rad.git
```

Comment:
For now, I'm requiring dev versions of `roman_datamodels` and `rad`. We can minpin to the next release whenever it's available.